packages/destination-actions/src/destinations/pipedrive/createUpdateActivity/__tests__/__snapshots__/snapshot.test.ts.snap

```diff
@@ -0,0 +1,15 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Testing snapshot for Pipedrive's createUpdateActivity destination action: all fields 1`] = `""`;
+
+exports[`Testing snapshot for Pipedrive's createUpdateActivity destination action: required fields 1`] = `""`;
+
+exports[`Testing snapshot for Pipedrive's createUpdateActivity destination action: required fields 2`] = `
+Headers {
+  Symbol(map): Object {
+    "user-agent": Array [
+      "Segment (Actions)",
+    ],
+  },
+}
+`;
```

Comment:
What's up here? What is this snapshot generated from ?

Reply:
https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#snapshot-testing

The `yarn jest --testPathPattern='./packages/destination-actions/src/destinations/<DESTINATION SLUG>' --updateSnapshot` command